joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1945,4 +1936,37 @@ class AstCreator(filename: String, global: Global)
   def astForYieldWithOptionalArgumentPrimaryContext(ctx: YieldWithOptionalArgumentPrimaryContext): Seq[Ast] = {
     astForYieldWithOptionalArgumentContext(ctx.yieldWithOptionalArgument())
   }
+
+  def resolveRequireOrLoadPath(argsAst: Seq[Ast]): Seq[Ast] = {
+    val importedNode =
+      argsAst.head.nodes.filter(node => node.isInstanceOf[NewLiteral]).head.asInstanceOf[NewLiteral]
+    val pathValue = importedNode.code.replaceAll("'", "").replaceAll("\"", "")
+    val result = pathValue match {
+      case path if Files.isRegularFile(Paths.get(path)) =>
+        path
+      case path if Files.isRegularFile(Paths.get(path + ".rb")) =>
+        s"${path}.rb"
+      case _ =>
+        pathValue
+    }
+    packageStack.append(result)
+    astForImportNode(importedNode.code)
+  }
+
+  def resolveRelativePath(currentFile: String, argsAst: Seq[Ast]): Seq[Ast] = {
+    val importedNode =
+      argsAst.head.nodes.filter(node => node.isInstanceOf[NewLiteral]).head.asInstanceOf[NewLiteral]
+    val pathValue        = importedNode.code.replaceAll("'", "").replaceAll("\"", "")
+    val updatedPath      = if (pathValue.endsWith(".rb")) pathValue else s"$pathValue.rb"
+    val currentDirectory = new java.io.File(currentFile).getParentFile
```

Comment:
Could you preferably use the `better.files.File` API? It is more idiomatic Scala style